CHARTER.md

```diff
@@ -0,0 +1,71 @@
+- allow a network of experts to share information about gaps and requirements for building editing frameworks on the web.  
+
+The specifications the Editing Taskforce is producing may be driven to the spec level directly by its members within the Web Applications Working Group, or be transferred to another, more appropriate working group after initial collection of requirements. Furthermore, the group defines its success as when it at the very least, produces a recommendation with well defined requirements that result in enhancing existing text editing related specification drafts or helps design new ones.
+
+The TF is expected to work on the following efforts until the end of the charter but it is not meant to be an exhaustive list:
+
+- [Async Clipboard API](https://bugs.chromium.org/p/chromium/issues/detail?id=931839) 
+- [ContentEditableDisabled](http://w3c.github.io/editing/contentEditableDisabled.html) - ability to disable system UI 
+- [EditContext API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/EditContext/explainer.md) 
+- [Highlight API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/highlight/explainer.md) 
+- Native Selection and Caret behaviors 
+- [SpellChecker API](https://github.com/w3c/editing/issues/166) 
+- [SIP Policy](https://github.com/whatwg/html/issues/4876) 
+- [Input Events](https://www.w3.org/TR/input-events-1/) 
+- [ContentEditable](https://w3c.github.io/contentEditable/) 
+
+Deliverables are expected to eventually be shipped as standards recommendation by the Web Applications Working Group, CSS Working Group or any other group where a draft specification may fall into.
+
+### 3. Participation
+
+This task force is designed to make participation relatively easy for people not currently involved in the standards process (typically, developers of editing tool libraries) who may not be amenable to signing up to a large in scope discussions that can be found in, for instance, the main Web Applications Working Group.
+
+The group encourages questions, comments and all technical discussions on its public [mailing list](https://lists.w3.org/Archives/Public/public-editing-tf/) and [document repositories](https://github.com/w3c/editing).
+
+Participants are reminded of the  [W3C's Code of Ethical Conduct](https://www.w3.org/Consortium/cepc/).
+
+ 
+### 4. Decision Policy
+
+Editing Task Force group will seek to make decisions when there is consensus. In cases where there is a need to formally produce a group resolution about a particular issue, its chair(s) will put a question about the issue to the group and gather responses (including any formal objections to formally publishing a document into the appropriate WG); then, after due consideration of all the responses, the Chair(s) will record a group resolution (possibly after a formal vote and also along with responding to any formal objections). 
```

Comment:
```suggestion
The Editing Task force operates under the WebApps Working Group's [decision policy](https://www.w3.org/2019/05/webapps-charter.html#decisions) and adheres to the WebApps WG [working mode](https://www.w3.org/2019/05/webapps-charter.html#working-mode).
```

Reply:
Sounds good. I would add that the TF's Chairs (or coordinators, depending on what we end up calling them) are generally expected to fulfill the role of the Chair listed in the decision policy and working mode, by delegation from the WG chairs (or if they're not, and we expect the actual WG chairs to be the only ones doing that, be clear about it too).

Reply:
Just to clarify this section. These decisions are not about whether something should become a spec draft but rather a shape of a proposal or a specific approach before we propose it to the appropriate WG.

Reply:
Ok, make sure that's clear when this is updated.